dynamo/preprocessing/PreprocessWorker.py

```diff
@@ -0,0 +1,34 @@
+from dynamo.dynamo_logger import main_info
+from dynamo.preprocessing.gene_selection_utils import log1p
+from anndata import AnnData
+
+from dynamo.tools.connectivity import neighbors as default_neighbors
+
+
+class PreprocessWorker:
+    def __init__(
+        self, normalize_by_cells_function=None, filter_genes_function=None, use_log1p=True, neighbors=default_neighbors
+    ) -> None:
+        self.normalize_by_cells = normalize_by_cells_function
+        self.filter_genes = filter_genes_function
+        self.use_log1p = use_log1p
+        self.log1p = log1p
+        self.neighbors = neighbors
```

Comment:
neighbors doesn't belong to this step. it is in tools by tl.moments which calls tl.neighbors